src/Microsoft.AspNet.Server.Kestrel/KestrelEngine.cs

```diff
@@ -91,16 +91,37 @@ public void Dispose()
 
         public IDisposable CreateServer(string scheme, string host, int port, Func<Frame, Task> application)
         {
-            var listeners = new List<Listener>();
+            var listeners = new List<IDisposable>();
 
             try
             {
+                var pipeName = (Libuv.IsWindows ? @"\\.\pipe\kestrel_" : "/tmp/kestrel_") + Guid.NewGuid().ToString("n");
```

Comment:
I've verified that both pipeName's work on OS X. I think the logic where it defaults to "/tmp/..." for non-Windows OSs is good.